go/tools/test_init/test_init.go

```diff
@@ -0,0 +1,29 @@
+package test_init
+
+// This package must have no deps beyond Go SDK.
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// This initializer runs before any user packages.
+func init() {
```

Comment:
It looks like this ignores the `rundir` attribute. The working directory shouldn't always be `TEST_SRCDIR`: `go_test` needs to be able to choose the directory.

There are a couple ways to pipe that value in here. To avoid generating a `testinit` package for each binary, you could define a global `RunDir string`, then set it by passing `-X github.com/bazelbuild/rules_go/go/tools/testinit.RunDir=somedir` to the linker.

Reply:
done